gamestonk_terminal/config_plot.py

```diff
@@ -1,5 +1,19 @@
 PLOT_DPI = 100
 
+# Backend to use for plotting.  After using the bt library, which somehow changes the
+# backend to 'agg' it raises an error.  Set this to avoid.
+"""
+More information:
+https://matplotlib.org/stable/tutorials/introductory/usage.html#the-builtin-backends
+
+Common ones:
+module://backend_interagg   : This is what pycharm defaults to in Scientific Mode
+MacOSX   : Mac default.  Does not work with backtesting
+tkAgg : This uses the tkinter library.
+Qt5Agg  : This requires the PyQt5 package is installed
+"""
+backend = None
```

Comment:
Maybe add this in description of PR " After using the bt library, which somehow changes the
# backend to 'agg' it raises an error.  

Could we re-write this as, and put it at the bottom? 

```
# Backend to use for plotting
backend = None
# Examples:
# "module://backend_interagg" - This is what pycharm defaults to in Scientific Mode
# "MacOSX" - Mac default.  Does not work with backtesting
# "tkAgg" - This uses the tkinter library.
# "Qt5Agg" - This requires the PyQt5 package is installed
# See more: https://matplotlib.org/stable/tutorials/introductory/usage.html#the-builtin-backends
```

Reply:
I can make it like that (should I comment each line or just keep the doctoring format?)

And I will update the PR description now.

Reply:
As you have it is good! Once you fix the linter, feel free to merge this!